### PR TITLE
Ionic Release Notes and Highlights

### PR DESCRIPTION
**Pull request by azeey**

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
+- [Make the single argument constructor of `Subprocess` inherit the env](https://github.com/gazebosim/gz-utils/pull/113)
+- [bazel: build and test subprocess functionality](https://github.com/gazebosim/gz-utils/pull/123)
+- [bazel:Add license checking support](https://github.com/gazebosim/gz-utils/pull/108)
+- [Add package.xml](https://github.com/gazebosim/gz-utils/pull/125)
+- [Require cmake version 3.22.1](https://github.com/gazebosim/gz-utils/pull/132)
 
-## Breaking Changes
 
-## Documentation
+## gazebosim/gz-physics:
```
Comment (scpeters) [inline release_notes.md:72]: what is the ordering strategy for the package release notes? it doesn't seem to be topological or alphabetical

Comment (azeey) [reply release_notes.md:72]: I'm thinking alphabetical since it's the easiest.

Comment (azeey) [reply release_notes.md:72]: I think that's what did for Harmonic https://github.com/gazebosim/gz-harmonic/blob/main/release_notes.md

File: highlights.md [hunk 2]
```diff
@@ -2,9 +2,178 @@
...
+  - In prior Gazebo versions, if a user specified a server plugin at the
+    world level, it would override all default plugins. This required
+    users to have in-depth knowledge and specify every plugin necessary
+    for a simulation to run correctly. Omitting critical plugins would
+    lead to unexpected simulation behavior, causing confusion,
+    particularly for new Gazebo users.
+
+  - Gazebo Ionic addresses this issue by loading default plugins even when
+    users specify additional plugins. Gazebo Ionic also introduces
+    `<gz:policy>` settings to revert this behavior to how it functioned in
```
Comment (scpeters) [inline highlights.md]: we should either add a snippet including `<gz:policy>` or provide a link to documentation about it

**Comment by scpeters** [comment]
should we mention `package.xml` files in the highlights?

File: release_notes.md (hunk 1 above)
Comment (azeey) [reply release_notes.md:72]: Fixed in https://github.com/gazebosim/gz-ionic/pull/4/commits/3fc4ff3027a7041166108686a69b017d05c446f7

File: highlights.md (hunk 2 above)
Comment (azeey) [reply highlights.md]: Added links in https://github.com/gazebosim/gz-ionic/pull/4/commits/5208be50493cdce8db0b73b19eb6a5f1e409f833

**Comment by azeey** [comment]
> Add highlight about package.xml

https://github.com/gazebosim/gz-ionic/pull/4/commits/527c9f41d7421fab8a9f774b60890667aac499a0

**Review by caguero — APPROVED** [review]

**Pull request merged**